### PR TITLE
Fix: EulaText string format error in EulaDialog

### LINKED ISSUE #299: Error in Eula Dialog string formatting
**Summary:** 
Error in Eula Dialog string formatting

**Device and Android version:** 
8.0.0

**Screenshots:** 
Screenshot (190)

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #299 
- [x] Run the unit tests with `gradlew app:connectedGmsDebugAndroidTest` to make sure you didn't break anything (with an emulator running or phone connected)

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by jddeep** [comment]
@jaydeetay Please review.

File: app/src/main/java/com/google/android/stardroid/activities/dialogs/EulaDialogFragment.java [hunk 1]
```diff
@@ -56,7 +56,8 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
     TextView apologyTextView = (TextView) view.findViewById(R.id.language_apology_box_text);
     apologyTextView.setText(formattedApologyText, TextView.BufferType.SPANNABLE);
 
-    String eulaText = String.format(parentActivity.getString(R.string.eula_text), getVersionName());
+    String eulaText = String.format(parentActivity.getString(R.string.eula_text) +
```
Comment (jaydeetay) [inline app/src/main/java/com/google/android/stardroid/activities/dialogs/EulaDialogFragment.java]: This isn't quite right. The version number is supposed to be interpolated into the text.
That said....due to refactorings of the text it looks like there's no longer a placeholder for it in the EULA text.  That's not the end of the world, but we could just remove the String.format and getVersionName altogether.

Comment (jddeep) [reply app/src/main/java/com/google/android/stardroid/activities/dialogs/EulaDialogFragment.java]: @jaydeetay So you want `eula_text` to be concatenated with the version name ? And get assigned to `eulaText` right?

**Comment by jddeep** [comment]
@jaydeetay Changes made, Please review.

File: app/src/main/java/com/google/android/stardroid/activities/dialogs/EulaDialogFragment.java (hunk 1 above)
Comment (jaydeetay) [reply app/src/main/java/com/google/android/stardroid/activities/dialogs/EulaDialogFragment.java]: Not really. The original text would have had a placeholder in it. Something like "EULA for version: %s.  Blah blah blah...".  So concatenating isn't the right thing to do. Over time it looks like the version number has been removed from the text (or perhaps it wasn't present) so the String.format isn't doing anything.  I think it's harmless, but it's also redundant.

**Pull request merged**